e2etest/utils.go

```diff
@@ -76,3 +136,31 @@ func tweakRCPort() func(rc *v1.ReplicationController) {
 	}
 	return nil
 }
+
+func updateConfigMapObjs(cs clientset.Interface, dataModifier func(m *configFile)) error {
```

Comment:
I think the whole config parsing stuff deserves a separate package.
We should aim to keep the utils.go file slim / remove it, not adding new stuff to it.
https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common